spec/generate_pairs_spec.rb

```diff
@@ -46,12 +46,13 @@ module MakersToolbelt
     describe 'result' do
       it 'contains a JSON formatted one-factorization of the names in the file' do
         names = GeneratePairs.load_names(READ_PATH)
+
+        allow(GeneratePairs).to receive(:load_names).and_return(double(one_factorize: double(shuffle: names.one_factorize)))
```

Comment:
yeah I would normally just stub `File.read_lines` instead to avoid stubbing the object under test